### PR TITLE
[PIP-82] [pulsar-broker] Rename RGUsageMTAggrWaitForAllMesgs, so that…

### PR DESCRIPTION
**Pull request by kaushik-develop**
… it gets included in mvn test runs

    - note: the test will fail until the https://github.com/apache/pulsar/issues/11289 regression is fixed

### Motivation

*Explain here the context, and why you're making that change. What is the problem you're trying to solve.*

Rename the class RGUsageMTAggrWaitForAllMesgs by adding a "Test" suffix, so that it is picked up in Pulsar CI tests. [The test would have flagged the issue mentioned in https://github.com/apache/pulsar/issues/11289]

### Modifications

Renamed the class, and the file-name; added comments, and addressed build warnings in the class.

### Verifying this change

- [ ] Make sure that the change passes the CI checks.
- The CI test will fail on this change, until Issue 11289 is fixed. 

*(Please pick either of the following options)*

This change is already covered by a test:
mvn test -Dtest=org.apache.pulsar.broker.resourcegroup.RGUsageMTAggrWaitForAllMesgsTest  -pl pulsar-broker

### Does this pull request potentially affect one of the following parts:

*If `yes` was chosen, please highlight the changes*

  - Dependencies (does it add or upgrade a dependency): (no)
  - The public API: (no)
  - The schema: (no)
  - The default values of configurations: (no)
  - The wire protocol: (no)
  - The rest endpoints: (no)
  - The admin cli options: (no)
  - Anything that affects deployment: (no)

### Documentation

  - Does this pull request introduce a new feature? (no)
  - If yes, how is the feature documented? (not applicable)
  - If a feature is not applicable for documentation, explain why? (this PR doesn't introduce a new feature)
  - If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

File: pulsar-broker/src/test/java/org/apache/pulsar/broker/resourcegroup/RGUsageMTAggrWaitForAllMesgsTest.java [hunk 1]
```diff
@@ -57,7 +57,7 @@
 // The tenants and namespaces in those topics are associated with a set of resource-groups (RGs).
 // After sending/receiving all the messages, traffic usage statistics, and Prometheus-metrics
 // are verified on the RGs.
-public class RGUsageMTAggrWaitForAllMesgs extends ProducerConsumerBase {
+public class RGUsageMTAggrWaitForAllMesgsTest extends ProducerConsumerBase {
```
Comment (aahmed-se) [inline pulsar-broker/src/test/java/org/apache/pulsar/broker/resourcegroup/RGUsageMTAggrWaitForAllMesgsTest.java:60]: You need to assign these to one of the existing test groups used amoung broker tests.

Comment (kaushik-develop) [reply pulsar-broker/src/test/java/org/apache/pulsar/broker/resourcegroup/RGUsageMTAggrWaitForAllMesgsTest.java:60]: AFAICT, it runs under the group "other".

**Comment by codelipenghui** [comment]
The pr had no activity for 30 days, mark with Stale label.

**Comment by tisonkun** [comment]
The original file is missing. Closed as invalid.

**Pull request closed without merging**